.github/workflows/CI.yml

```diff
@@ -55,10 +55,10 @@ jobs:
         run: yarn check-types
 
       - name: Run unit tests
-        run: yarn unit-test
+        run: timeout 15m yarn unit-test
 
       - name: Run API tests
-        run: yarn test-api
+        run: timeout 15m yarn test-api
```

Comment:
does this mean that all of our CI operations will now take 15m a pop?

Reply:
This means that if our CI tests take longer than 15m it will throw an error.

See https://man7.org/linux/man-pages/man1/timeout.1.html for description

And https://github.com/hicommonwealth/commonwealth/actions/runs/4256371747/jobs/7405175475 in practice (although doesn't error because tests only take 5 mins)